docker-compose.greenlight.yml

```
@@ -0,0 +1,24 @@
+version: '3.6'
+
+services:
+  greenlight:
+    container_name: greenlight-v2
+    image: bigbluebutton/greenlight:v2
+    env_file: .env
+    environment:
+      DB_ADAPTER: postgresql
+      DB_HOST: postgres
+      DB_NAME: greenlight
+      DB_USERNAME: postgres
+      DB_PASSWORD: password
+      BIGBLUEBUTTON_ENDPOINT: https://${DOMAIN}/bigbluebutton/api/
+      BIGBLUEBUTTON_SECRET: ${SHARED_SECRET}
+      SECRET_KEY_BASE: ${RAILS_SECRET}
+    ports:
+      - 10.7.7.1:5000:80
+  postgres:
+    image: postgres:12
+    environment:
+      POSTGRES_DB: greenlight
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
```

It seems to me that we need a volume here in order not to loose all accounts and rooms when issuing `docker-compose ... down`.
So we would need something along the lines of 
```suggestion
      POSTGRES_PASSWORD: password
    volumes:
      - "./greenlight-postgresql/:/var/lib/postgresql/data"
```
or make the location of the volume (possibly a docker volume would be preferred) configurable. Or put it into some subdirectory of the existing volume `bigbluebutton`. Or at least mention in the installation instructions that a customization is required here (I happily created users after installation and was then somewhat puzzled that all users had gone).

---

Thanks for the suggestion and sorry for the troubles.

I've added now a persistent folder with some upgrading advises: https://github.com/bigbluebutton/docker/pull/43/commits/03d682fb5b2c1ede34dbf9e68262588878d46d61